_posts/2017/2017-04-06-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,75 @@
+authors: ["Benjamin, Bastian"]
+---
+
+The [JavaLand 2017](https://www.javaland.eu/en/javaland-2017/) conference took place for the 4th time in the famous [Phantasialand](http://www.phantasialand.de/en/) holiday park in Brühl, near Cologne.
+The program consisted of [talks](https://programm.javaland.eu/2017/schedule.html) on the frist two days and [in-depth workshops](https://www.javaland.eu/de/programm/schulungstag) at the last and third day.
+
+Here are some random facts so that you can imagine the dimension of the largest community conference for Java in Europe:
+
+- 1600 visitors, 14 different locations in the holiday park
+- 11 parallel talk tracks, 8 session slots each day
+- 8 in-depth workshops which over 8 hours each
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="und" dir="ltr"><a href="https://twitter.com/hashtag/welcome?src=hash">#welcome</a> to <a href="https://twitter.com/hashtag/JavaLand?src=hash">#JavaLand</a> 💙 <a href="https://t.co/wX37hzFVXM">pic.twitter.com/wX37hzFVXM</a></p>&mdash; JavaLandConf (@JavaLandConf) <a href="https://twitter.com/JavaLandConf/status/846628341209878528">March 28, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Talks
+
+We visited several talks and three workshops:
+
+## Reactive Programming
+
+[A Reactive Take on Pokémon GO](https://programm.javaland.eu/2017/talk.html#talk?talkId=529316) by [Riccardo Lippolis](https://www.linkedin.com/in/rlippolis) and [Bas Knopper](https://twitter.com/BWknopper) was all about [Spring Reactive](http://docs.spring.io/spring-framework/docs/5.0.0.M1/spring-framework-reference/html/web-reactive.html). After a quick introduction he showed how to build a reactive program for smartphones.
+He extended the observer pattern and added push-based streams, which can be used when it is unknown when data is available.
+At its core the [Spring Reactor](fully non-blocking foundation with efficient demand management), a fully non-blocking foundation with efficient demand management, is used.
+Another important role plays [Flux](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html) which is similar to the [Java 8 Stream API](https://docs.oracle.com/javase/8/docs/api/java/util/stream/package-summary.html).
+A new paradigm in Spring Land will be released in June and is called [Spring Web Flux](https://spring.io/blog/2017/03/15/spring-tips-the-spring-web-flux-reactive-client).
+See [this video](https://www.youtube.com/watch?v=leZdgr-O4LE) by [Josh Long](https://twitter.com/starbuxman) for more details.
+
+## Migrating to Java 9
+
+In [this talk](https://programm.javaland.eu/2017/talk.html#talk?talkId=529430) by [Sander Mak](https://twitter.com/sander_mak), who is currently writing a book with [Paul Bakker](https://twitter.com/pbakker) about [Java 9 Modularity](https://twitter.com/javamodularity), explained in this talk the essential steps to prepare for the next Java major release.
```

Comment:
In this talk by Sander Mak, [...], he explained essential steps to prepare for the next major Java release.